controllers/packetmachine_controller.go

```diff
@@ -238,8 +239,14 @@ func (r *PacketMachineReconciler) reconcile(ctx context.Context, machineScope *s
 		createDeviceReq.ExtraTags = tags
 
 		dev, err = r.PacketClient.NewDevice(createDeviceReq)
-		if err != nil {
-			errs := fmt.Errorf("failed to create machine %s: %v", machineScope.Name(), err)
+
+		switch {
+		// TODO: find a better way than parsing the error messages for this.
```

Comment:
I'm not overly happy with this approach, but I wasn't able to find a unique error code that we could match against and this substring appears to be unique from the API for this error. This is the same approach taken in the autoscaler integration: https://github.com/kubernetes/autoscaler/pull/3483/files#diff-364a854e7bf33e867868548a8220972f41ca9cd66260a875d728f20c9500a13cR559-R562